### ISSUE TITLE
Improvement of Compare Page in Website

### ISSUE THREAD

**Issue by aditya-chari25** [MAVRICK-1/Nest-Ondc#101]
**Is your feature request related to a problem? Please describe.**
The Compare Page in the website is very basic and I am focused on improving the website by implementing the below ideas.

**Describe the solution you'd like**
1. In the compare page between the items selected we can display the items vertically since for comparison will be easier.
2. In every item only the price ,description, by whom and distance will be present, in tabular column for easier comparison
3. And while adding we have to update the no. of comparisons above the Compare icon present in top right.
basically the vertical description can be like this below.

![example-comp](https://github.com/MAVRICK-1/Nest-Ondc/assets/84094715/f5fc7704-391a-49f4-aa53-f2c9655c73b1)

**Describe alternatives you've considered**
An additional feature can be cross-topic item comparison that is, if we select items from electronics and groceries
we can create comparisons among two different topics which are present.

**Additional context**

**Comment by aditya-chari25** [comment]
@MAVRICK-1 pls assign me this issue under SSOC'24

**Comment by akbatra567** [comment]
PR already merged.
